GPX/GPXRoot.h

```diff
@@ -39,6 +39,9 @@
 /** Metadata about the file. */
 @property (strong, nonatomic) GPXMetadata *metadata;
 
+/** Keywords for the GPX file. */
```

Comment:
Any recommendation as to how they should be delimited? Should we rather pass an array, and concat using a known delimiter internally?